app/src/screens/MapScreen.js

```diff
@@ -115,10 +117,49 @@ function MapScreen(): React.Node {
     [unfilteredLayers],
   );
 
-  const rangeFactor = useMemo(
-    () => (-1 * 255) / (maxRssiToDisplay - minRssiToDisplay),
-    [maxRssiToDisplay, minRssiToDisplay],
-  );
+
+  const signalBuckets = [
+    {
+      color: COLORS.heat10,
+      maxValue: -60,
+    },
+    {
+      color: COLORS.heat20,
+      maxValue: -65,
+    },
+    {
+      color: COLORS.heat30,
+      maxValue: -70,
+    },
+    {
+      color: COLORS.heat40,
+      maxValue: -75,
+    },
+    {
+      color: COLORS.heat50,
+      maxValue: -80,
+    },
+    {
+      color: COLORS.heat60,
+      maxValue: -85,
+    },
+    {
+      color: COLORS.heat70,
+      maxValue: -90,
+    },
+    {
+      color: COLORS.heat80,
+      maxValue: -95,
+    },
+    {
+      color: COLORS.heat90,
+      maxValue: -100,
+    },
+    {
+      color: COLORS.heat100,
+      maxValue: -105,
+    },
+  ];
```

Comment:
I'd put this in the SignalStrengthLegend component, the utility file, or in its own file. MapScreen is already super large and complicated. We can just import it from there.

Also if possible, I'd separate the values from the colormaps, and join them using a function. This would allow being able to customize this later.

e.g. something like the following but more legible `(a, b) => a.map((k, i) => ({color:k, maxValue:b[i]]});`